packages/desktop-client/src/hooks/useLocalPref.ts

```diff
@@ -16,8 +16,11 @@ export function useLocalPref<K extends keyof LocalPrefs>(
 ): [LocalPrefs[K], SetLocalPrefAction<K>] {
   const [budgetId] = useMetadataPref('id');
 
-  return useLocalStorage<LocalPrefs[K]>(`${budgetId}-${prefName}`, undefined, {
-    deserializer: JSON.parse,
-    serializer: JSON.stringify,
-  });
+  const [value, setter] = useLocalStorage<LocalPrefs[K]>(
```

Comment:
`useLocalStorage` returns an array of 3

Reply:
Why not return the 3rd value too now and instead update the return type on line 16?

I think we're also missing the serializer & deserializer now.

Reply:
Good call, not sure what I was thinking but it's fixed now